docs/starting-selenoid-manually.adoc

```diff
@@ -107,7 +107,7 @@ docker run -d                                   \
 --name selenoid                                 \
 -p 4444:4444                                    \
 -v /var/run/docker.sock:/var/run/docker.sock    \
--v ./config/:/etc/selenoid/:ro              \
+-v $(pwd)/config/:/etc/selenoid/:ro              \
```

Comment:
@p4m-dev `$(pwd)` was removed intentionally in https://github.com/aerokube/selenoid/commit/69405aefa6caab1adcce0cc48d971aa0ba492692. I guess it should be `/your/directory/` for consistency

Reply:
Yeah, it makes sense